src/jp2image.cpp

```diff
@@ -517,9 +520,7 @@ namespace Exiv2
                                 if ( bPrint ) {
                                     out << " | iccLength:" << iccLength ;
                                 }
-
-                                DataBuf icc(iccLength);
-                                if ( bICC ) out.write((const char*)icc.pData_,icc.size_);
+                                if ( bICC ) out.write((const char*)data.pData_+pad,iccLength);
```

Comment:
[minor thing] I think we should discuss about the the style for the one-line if blocks in the code. 

I found this problematic when using code coverage tools, because we cannot know if the statement under the condition is exercised or not.

Anyways, this is something that we could fix later applying clang-format ;)

Reply:
I neither do like one-line if blocks because it also makes debugging less intuitive.